### PR TITLE
Coding style of Stack module

### PR DESCRIPTION
**Pull request by mihails-strasuns-sociomantic**

### CODE REVIEW

File: src/ocean/util/container/more/Stack.d [hunk 1]
```diff
@@ -1,317 +1,316 @@
...
-                  }
-               else
-                  {
-                  private uint     depth;
-                  private V[Size]  stack;
-                  }
+        Clear the stack
 
-        /***********************************************************************
+        Returns: pointer to itself for chaining calls
```
Comment (gavin-norman-sociomantic) [inline src/ocean/util/container/more/Stack.d:55]: Is `Returns: <whatever>` without a newline valid ddoc? (It's at least unusual for our style.)

Comment (nemanja-boric-sociomantic) [reply src/ocean/util/container/more/Stack.d:55]: it's valid ddoc, but still unusual for our style.

Comment (mihails-strasuns-sociomantic) [reply src/ocean/util/container/more/Stack.d:55]: It is not that uncommon:

```
$ grep -R -n "Returns: " ./src/ | wc -l
269
```

I have been using it in ocean for quite some time with both `Returns:` and `Throws:` to save some vertical space.

Comment (gavin-norman-sociomantic) [reply src/ocean/util/container/more/Stack.d:55]: Go on then. (Our coding style doc doesn't mention this either way, though by convention it's always included a newline, until you came along with your newfangled ideas.)

**Review by gavin-norman-sociomantic — APPROVED** [review]

**Comment by gavin-norman-sociomantic** [comment]
Red.

**Comment by mihails-strasuns-sociomantic** [comment]
Fixed D2 build

**Comment by mihails-strasuns-sociomantic** [comment]
Hm, codecov report shows some obviously covered lines as not covered. Gonna have a look.

**Comment by mihails-strasuns-sociomantic** [comment]
Will recheck after https://github.com/sociomantic-tsunami/ocean/pull/312 is merged

**Comment by mihails-strasuns-sociomantic** [comment]
Green with proper coding coverage

**Pull request merged**